src/main/java/practice/StreamPractice.java

```diff
@@ -59,7 +78,11 @@ public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
      * return the names of all cats whose owners are women from `femaleAge` years old inclusively.
      */
     public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(n -> n.getAge() >= femaleAge && n.getSex().equals(Person.Sex.WOMAN))
```

Comment:
```suggestion
                .filter(n -> n.getAge() >= femaleAge && n.getSex() == Person.Sex.WOMAN)
```
You can even use ==, read common mistakes